test/cmdlineTests/model_checker_engine_all/exit

```diff
@@ -0,0 +1 @@
+0
```

Comment:
I think that `exit` file is optional if the code is zero so you might remove it to have less clutter...

Reply:
`err` is optional too if there's no output on stderr.